### ISSUE TITLE
MQTT connection for CLI

### ISSUE THREAD

**Issue by WebSpider** [skodaconnect/myskoda#26]
Let's discuss this in an issue and keep it out of this PR please :)

_Originally posted by @Prior99 in https://github.com/skodaconnect/myskoda/pull/10#discussion_r1771275581_

**Comment by OBerghmans** [comment]
If I run the CLI I always get "Failed to connect to MQTT". I have not yet figured out whether it's something on my side, on the code side or just the MQTT server side.

However, I saw that quite some CLI commands don't really need the MQTT connection. Perhaps we can look into adjusting the CLI that it doesn't set up this connection if we're just going to make a API request and exit anyway.
That will prevent making unnecessary connections, but might make the CLI code a bit more complex.

Another option might be a "--disable-mqtt" flag, a bit more simple but less elegant.

Yesterday I commented out the mqtt part when preparing my PR as I didn't need the MQTT connection and the failure to connect prevented me from testing my changes.

**Comment by OBerghmans** [comment]
An Enyaq 85x, this is the output in the CLI info command (mqtt lines were still commented out :-) )
```
model: Enyaq Coupé
model id: 5ACJK4
model year: 2025
title: Škoda Enyaq Coupé
```

**Comment by OBerghmans** [comment]
The tcptraceroute works if I run it from a docker container. I just tried to run the CLI inside that container as well and that works too. Seems to be (my) windows related.

Anyway, apart from my issues, the goal of my comment was mainly the optimization of not running the MQTT if we don't need it. Like just now in my docker container, I ran the info command and it just connected and disconnected from MQTT without really needing it. Basically the same remark as @WebSpider made in that PR that led to this issue. 

We could add a flag on the myskoda class, with a default value of connecting to mqtt. From the CLI module we could then turn off the mqtt for commands that don't need it. That removes the unnecessary connection to mqtt and also all the related logs to this.
If you're open to the idea, I am willing to give it a shot and make a PR.
